### ISSUE TITLE
Viewing reports

### ISSUE THREAD

**Issue by ScottVinje** [spectralsun/emcit#36]
So far the app is looking good.  The main issue I see is that there is no narrative portion showing in the view reports portion of the app.  This is probably the most important field to have, it is what puts all of the other information in context.

**Comment by PamelaKinion** [comment]
What Scott is referring to (I believe, correct me if I'm wrong) is when you click on one entry in the search summary, the resulting report is a code block.  

I agree that this probably needs to be our #1 modification scheduled.  Ideally, it would be a complete summary of the entry in plain English.  Including the ID number of the reporter.  We should be able to print it or attach it to an email in a form that the receiver can read.

**Comment by slively** [comment]
@Pamela can you send me a link to an example of this?

**Comment by slively** [comment]
I talked to the team about this and what you are seeing is a placeholder page. If you look closely at the text you will notice it actually shows you all the data for that report. We'll have to take the time to create a page for a single report which should be pretty easy.
